### PR TITLE
docs: Update installation document to clarify sample data ingestion process

### PR DESCRIPTION
**Pull request by Alagappan**
### Summary of Changes
Updated installation guide to clarify the need for creating a virtual env in addition to having a docker image that runs Amundsen service. It was not clear that the virtual env and installing dependencies was only required for the sample data loader script we provide.

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [X] PR title addresses the issue accurately and concisely, including a title prefix.
- [X] PR includes a summary of changes.
- [X] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ The following instructions are for setting up a version of Amundsen using Docker
...
-4. Ingest dummy data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
+4. Ingest provided sample data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
 
-   * Change directory to the [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder) submodule.
-   * Run the following commands in the `amundsendatabuilder` upstream directory:
+   * In a separate terminal window, change directory to the [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder) submodule.
+   * `sample_data_loader` python script included in `examples/` directory uses _elasticsearch client_, _pyhocon_ and other libraries. Install the dependencies in a virtual env and run the script by following the commands below:
    ```bash
     $ python3 -m venv venv
     $ source venv/bin/activate
```
Comment (feng-tao) [inline docs/installation.md:25]: thanks @Alagappan , but wouldn't L24-L25 suggested creating virtual env?

Comment (Alagappan) [reply docs/installation.md:25]: yes, the command work as expected. It wasn't clear why we need to install the dependencies again separately (and requirements.txt)  when we have docker container running already from step 3. Just updated the instructions to clarify we only need this step for sample script.

Comment (feng-tao) [reply docs/installation.md:25]: thanks

**Comment by jornh** [comment]
Great docs clarification @Alagappan!

This should resolve https://github.com/lyft/amundsen/issues/577 as per discussion in https://amundsenworkspace.slack.com/archives/CHQERT0D7/p1595605193101400?thread_ts=1595573987.100600&channel=CHQERT0D7&message_ts=1595605193.101400

Cc: @deyerly

**Pull request merged**